web/components/admin/Toolbar/index.js

```diff
@@ -67,6 +85,19 @@ class Toolbar extends React.Component {
             src={areFiltersApplied ? filterIconSelected : filterIcon}
             alt="filter button" />
         </div>
+        <TextInput
```

Comment:
You should be able to fix the alignment and height of the search box by adding something like the following to the `.toolbar` class (in `index.sass` for the Toolbar)
```
  .text-input
    input
      margin-top: 0
      height: 48px
```

Reply:
Worked perfectly thank you!